### DISCUSSION TITLE
Support FFZ user-limited emotes

### DISCUSSION THREAD

**Discussion by atrigent** [Chatterino/chatterino2#4444] [Ideas] [answered]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

FFZ supports emotes that only some users can use in their messages. The endpoint returns data like this:

```json
{
  "default_sets": [
    3
  ],
  "sets": {
    "3": {
      "id": 3,
      "_type": 1,
      "icon": null,
      "title": "Global Emotes",
      "css": null,
      "emoticons": [
        ...
      ]
    },
    "1532818": {
      "id": 1532818,
      "_type": 1,
      "icon": null,
      "title": "Subwoofer Emote Effects",
      "css": null,
      "emoticons": [
        ...
      ]
    }
  },
  "user_ids": {
    "1532818": [
      ...
    ]
  }
}
```

This means that only certain users (listed in `user_ids`) can use the emotes in the "Subwoofer Emote Effects" emote set (set id 1532818). "Subwoofer" is a support tier for FFZ, so these are people that have paid FFZ for access to extra emotes. I think this meshes with Chatterino's goal to help support the services that it makes use of.

This specific emote set makes heavy use of FFZ's "emote effects", which Chatterino also doesn't support. There is already a discussion of this at #4429.

### OS and Chatterino Version

2.4.2

**Comment by Felanbird** [comment, marked as answer]
It is very unlikely we will add these. As you may or may not be aware, BTTV also has a paid-only feature for emotes, which we also do not support.

It is not the intention of Chatterino to display emotes only _some_ users can use, with the exception of Twitch subscription emotes, as it's literally their website.
